gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/rebind/GinjectorGenerator.java

```diff
@@ -129,9 +135,43 @@ private void findAllPresenters(PresenterDefinitions presenterDefinitions) throws
             } else if (type.isAnnotationPresent(DefaultGatekeeper.class)) {
                 presenterDefinitions.addGatekeeper(type);
             }
+
+            for (Class<? extends Annotation> annotation : annotations) {
+                if (type.isAnnotationPresent(annotation)) {
+                    presenterDefinitions.addStandardPresenter(type);
+                    break;
+                }
+            }
         }
     }
 
+    private List<Class<? extends Annotation>> findGinjectorAnnotations() throws UnableToCompleteException {
+        List<Class<? extends Annotation>> annotations = new ArrayList<Class<? extends Annotation>>();
+
+        List<String> values = findConfigurationProperty(GIN_GINJECTOR_ANNOTATION).getValues();
+        if (values.size() > 0) {
```

Comment:
I agree. I just copy piece of code from addExtensionInterfaces and it has similar redundand check...